lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -90,11 +118,10 @@
         }
     },
     "variables": {
-        "virtual_network_name": "lisa-virtualNetwork",
-        "vnet_id": "[resourceId('Microsoft.Network/virtualNetworks/', variables('virtual_network_name'))]",
+        "vnet_id": "[resourceId('Microsoft.Network/virtualNetworks/', parameters('virtual_network_name'))]",
         "node_count": "[length(parameters('nodes'))]",
         "availability_set_name": "lisa-availabilitySet",
-        "subnet_prefix": "lisa-subnet-"
+        "existing_subnet_ref": "[if(parameters('use_existing_virtual_network'), resourceId(parameters('virtual_network_resource_group'), 'Microsoft.Network/virtualNetworks/subnets', parameters('virtual_network_name'), parameters('subnet_prefix')), '')]"
```

Comment:
If `use_existing_virtual_network` is false, how the default virtual network and subnet prefix are applied?